static/reference/actionscript/3.0/TimeZone.html

```diff
@@ -458,101 +458,101 @@
 
-		 </td>
-</tr>
-</table>
-<p></p>
-<span class="label">Returns</span>
-<table border="0" cellspacing="0" cellpadding="0">
-<tr>
+		 </td>
+</tr>
+</table>
+<p></p>
+<span class="label">Returns</span>
+<table border="0" cellspacing="0" cellpadding="0">
+<tr>
 <td width="20"></td><td><code><a href="int.html">int</a></code> &mdash; 
                         The offset, in milliseconds, between GMT and local time at the given date.
 
 		 
 		 
                         
-                     </td>
-</tr>
-</table>
-</div>
-<a name="getTimeZone()"></a>
-<table cellspacing="0" cellpadding="0" class="detailHeader">
-<tr>
-<td class="detailHeaderName">getTimeZone</td><td class="detailHeaderParens">()</td><td class="detailHeaderType">method</td><td class="detailHeaderRule">&nbsp;</td>
-</tr>
-</table>
-<div class="detailBody">
-<code>public static function getTimeZone(timeZoneName:<a href="String.html">String</a>):<a href="TimeZone.html">TimeZone</a></code>
-<p></p>
-<table border="0" cellspacing="0" cellpadding="0">
-<tr>
-<td valign="top" style="white-space:nowrap"><b>Language version:&nbsp;</b></td><td>ActionScript 3.0
-		 </td>
-</tr>
-</table>
-<table border="0" cellspacing="0" cellpadding="0">
-<tr>
-<td valign="top" style="white-space:nowrap"><b>Runtime version:&nbsp;</b></td><td>AIR&nbsp;50
-
-		</td>
-</tr>
-</table>
+                     </td>
+</tr>
+</table>
+</div>
+<a name="getTimeZone()"></a>
+<table cellspacing="0" cellpadding="0" class="detailHeader">
+<tr>
+<td class="detailHeaderName">getTimeZone</td><td class="detailHeaderParens">()</td><td class="detailHeaderType">method</td><td class="detailHeaderRule">&nbsp;</td>
+</tr>
+</table>
+<div class="detailBody">
+<code>public static function getTimeZone(timeZoneName:<a href="String.html">String</a>):<a href="TimeZone.html">TimeZone</a></code>
+<p></p>
+<table border="0" cellspacing="0" cellpadding="0">
+<tr>
+<td valign="top" style="white-space:nowrap"><b>Language version:&nbsp;</b></td><td>ActionScript 3.0
+		 </td>
+</tr>
+</table>
+<table border="0" cellspacing="0" cellpadding="0">
+<tr>
+<td valign="top" style="white-space:nowrap"><b>Runtime version:&nbsp;</b></td><td>AIR&nbsp;50
+
+		</td>
+</tr>
+</table>
 <p></p><p>
 
 		 Obtains a TimeZone object using a string that is found in the list of <code>availableTimeZoneNames</code>.
 
 		 Pass in a null string to get the TimeZone object representing the current location of the device.
 
 		 
-		 </p><span class="label">Parameters</span>
-<table border="0" cellspacing="0" cellpadding="0">
-<tr>
+		 </p><span class="label">Parameters</span>
+<table border="0" cellspacing="0" cellpadding="0">
+<tr>
 <td width="20px"></td><td><code><span class="label">timeZoneName</span>:<a href="String.html">String</a></code> &mdash; The name of the time zone to retrieve, or null or the current device time zone.
```

Comment:
"or null or the current device time zone."
May be typo and need to be changed to:
"or null for the current device time zone."

Reply:
@ajwfrost Do you want to resolve this issue in this PR or do you want to do another at a later point?

Reply:
@marchbold have just found out how to do that directly... so I've edited the file here (and the AS3 source file in our Adobe repository), hopefully this works!

thanks